service/src/integrationTest/resources/fixtures/initiate-case.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I believe we could use `core-case-data-store-api/callback-request.json` and remove `initiate-case.json`. Having one fixture should make maintenance easier and we can always introduce new one when really needed.

Reply:
Wired, what is the error?